### DISCUSSION TITLE
Bro, I made a bomb explode from a far distance and when I returned to the exploded area, well, the game closed and I could hardly get out of that area, do you know any solution?

### DISCUSSION THREAD

**Discussion by Geogit12** [HbmMods/Hbm-s-Nuclear-Tech-GIT#490] [Q&A]
help

**Comment by MartinTheDragon** [comment]
Perhaps your computer does not support the graphics for the mushroom cloud? A crash report would be nice.

**Reply by Geogit12** [reply]
No, I had already tried other bombs at a close distance and it worked fine, but when the bomb exploded at a distance that the chunks did not load the area of ​​the explosion, and when I returned to that area it crashed, and I had a hard time getting out of that area

**Reply by ncr-t** [reply]
Without a crash report we cannot help. Please provide one.

**Reply by Geogit12** [reply]
If I launch a "nuclear bomb" is it recommended that I have to be in that area to avoid failures and obviously "die"?

**Reply by MientusOfficial** [reply]
call 911 you are having stroke

**Reply by 9Valjew** [reply]
Give us a fucking crash report to work with

**Comment by TheForge129** [comment]
[image: image.png]
